extensions/README.md

```diff
@@ -10,6 +10,9 @@ _Draft Khronos extensions are not ratified yet._
 * [KHR_materials_common](Khronos/KHR_materials_common/README.md)
 * [KHR_technique_webgl](Khronos/KHR_technique_webgl/README.md)
 
+#### Vendor extensions
+* [AVR_texture_offset_tile](Vendor/AVR_texture_offset_tile/README.md)
```

Comment:
Should this be `AVR_` or `EXT_texture_transform`?  For `EXT_`, it should have multiple implementations.

For more info, see https://github.com/KhronosGroup/glTF/tree/master/extensions#promoting-extensions

Reply:
Good review, and thanks for the link. Changing prefix to MSFT, and updating the extensions readme.